examples/seg/deeplabv3/README.md

```diff
@@ -0,0 +1,176 @@
+
+<p align="center">
+  <img src="https://github.com/mindspore-lab/mindcv/assets/33061146/e1a17518-b19a-46f1-b28a-ec67cafa81be" width=700/>
+</p>
+<p align="center">
+  <em>Figure 2. DeepLabv3+ extends DeepLabv3 by employing a encoderdecoder structure [<a href="#references">2</a>] </em>
+</p>
+
+
+This example provides implementations of DeepLabV3 and DeepLabV3+ using backbones from MindCV. More details about feature extraction of MindCV are in [this tutorial](https://github.com/mindspore-lab/mindcv/blob/main/docs/en/how_to_guides/feature_extraction.md). Note that the ResNet in DeepLab contains atrous convolutions with different rates,  `dilated_resnet.py`  is provided as a modification of ResNet from MindCV, with atrous convolutions in block 3-4.
+
+## Quick Start
+
+### Preparation
+
+1. Clone MindCV repository by running
+
+   ```shell
+   git clone https://github.com/mindspore-lab/mindcv.git
+   ```
+2. Install dependencies as shown [here](https://mindspore-lab.github.io/mindcv/installation/)
+
+3. Prepare dataset
+
+- Download Pascal VOC 2012 dataset,  [VOC2012](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/) and Semantic Boundaries Dataset, [SBD](https://www2.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz).
+
+- Prepare training and test data list files with the path to image and annotation pairs. You could simply run `python examples/seg/deeplabv3/build_dataset/get_dataset_lst.py --data_root=[PATH TO DATA]` to generate the list files. The lines in list file should be like as follows:
+
+  ```
+  /[PATH..]/JPEGImages/2007_000032.jpg /[PATH..]/SegmentationClassGray/2007_000032.png
+  /[PATH..]/JPEGImages/2007_000039.jpg /[PATH..]/SegmentationClassGray/2007_000039.png
+  /[PATH..]/JPEGImages/2007_000063.jpg /[PATH..]/SegmentationClassGray/2007_000063.png
+  ......
+  ```
+
+- Convert training dataset to mindrecords by running  ``bulid_seg_data.py `` script.
+
+  ```shell
+  python examples/seg/deeplabv3/bulid_dataset/bulid_seg_data.py \
+  		--data_root=[root path of training data] \
+  		--data_lst=[path of data list file prepared above] \
+  		--dst_path=[path to save mindrecords] \
+  		--num_shards=8 \
+  		--shuffle=True
+  ```
+
+* In accord with paper, we train on *trainaug* dataset (voc train + SBD) and evaluate on *voc val* dataset.
+
+4. Backbone: download pre-trained backbone from MindCV, here we use [ResNet101](https://download.mindspore.cn/toolkits/mindcv/resnet/resnet101-689c5e77.ckpt).
+
+### Train
+
+Specify `deeplabv3`  or  `deeplabv3plus` in key word `model` in the config file.
+
+It is highly recommended to use **distributed training** for this DeepLabV3 and DeepLabV3+ implementation.
+
+For distributed training using **OpenMPI's `mpirun`**, simply run
+```shell
+cd mindcv  # change directory to the root of MindCV repository
```

Comment:
hi, didn't we in the root of project when preparing data? Assumption like "We are always in the `${PROJECT_ROOT}`" may be better